pygfx/geometries/_compat.py

```diff
@@ -36,6 +36,9 @@ def geometry_from_trimesh(mesh):
         # the coordinate origin is in the upper left corner, while the opengl coordinate
         # origin is in the lower left corner.
         # trimesh loads textures according to the opengl coordinate system.
+        if mesh.visual.uv is None:
```

Comment:
I don't know enough about trimesh to understand why `mesh.visual.uv is None` when the kind is `"texture"`. Do you know?

Reply:
I made a model with textures and no UV mapping in Blender, which triggers this problem.

Reply:
You are right, this is problem of Trimesh.